src/components/Empty.jsx

```diff
@@ -8,8 +9,9 @@ export const Empty = ({ t, emptyType }) => {
     <div>
       {emptyType === 'devices' &&
         <div className={styles['set-empty']}>
-          <h2>{ t('Empty.devices_title') }</h2>
-          <p>{ t('Empty.devices_text')}</p>
+          <h2>{t('Empty.devices.title')}</h2>
+          <p>{t('Empty.devices.text')}</p>
+          <p><a className={classNames('coz-btn', 'coz-btn--regular')} href={t('Empty.devices.link.href')}>{t('Empty.devices.link.text')}</a></p>
```

Comment:
@GoOz, there is not yet an accessibility helper for the "opens in a new window" title ?